pkg/apis/servicecatalog/v1alpha1/zz_generated.defaults.go

```diff
@@ -28,5 +28,31 @@ import (
 // Public to allow building arbitrary schemes.
 // All generated defaulters are covering - they call all nested defaulters.
 func RegisterDefaults(scheme *runtime.Scheme) error {
```

Comment:
I don't know how it works either. A lot of this got cribbed from upstream with some guidance from @pmorie. But yes... after adding the defaulters, code generation produced some diffs after its next run.